data/korath/korath outfits.txt

```diff
@@ -91,6 +92,32 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Liquid Sodium Cooler"
```

Comment:
Yeah, I really think it is.  Especially when you're running more than just a TPC, with the new heat-based turret weapons, on Korath hulls with their heat dissipation in the .4-.5 range, and your primary military opponent operates in the Ember Wastes.  Or for the upcoming city-ships which are trying to cram as many souls as possible on board while still being able to escape from danger.

A lot of work from a lot of people went in to the balancing of these stats.  We've been very careful not to step on the Coalition's toes; this doesn't outright outclass a cooling module, but offers a different balance point.  That, and at 34 tonnes, over 9000 cooling, and its constant demand, it's too big and expensive to be worthwhile unless you *are* running a TPC.

If you're running LRMs, Blue Suns, or even a DPC, use Coal or Ka'het cooling.  If you're running anything other than a TPC, for that matter, use Coal or Ka'het cooling.  Even when you have a TPC, in an otherwise cool ship, use Coal or Ka'het cooling.  If you've got a TPC on top of a lot of other heat problems, or are rutting additional plasma cores on top of your TPC, that's when the Liquid Sodium Cooler becomes an *option*.  Not necessarily the best one for captains mixing outfits from different factions, but better that piling on umpteen heat shunts.

As for the sprite, I really like the new version, so I'm not sure what's bothering you about it.